go/src/koding/kites/config/konfig.go

```diff
@@ -135,6 +136,7 @@ func NewKonfig(e *Environments) *Konfig {
 		KontrolURL:         Builtin.Endpoints.Kontrol,
 		KloudURL:           Builtin.Endpoints.Kloud,
 		TunnelURL:          Builtin.Endpoints.TunnelServer,
+		RemoteURL:          Builtin.Endpoints.RemoteAPI,
```

Comment:
> moreover we should add two URLs per serivce - one public (for koding users) and one private (for inter-communication)

if we have a "base" domain structure you can keep the paths separate

> I think we should keep the URLs separate in case we'd want in future to have each service in separate container / POD

in any case we will need a proxy in between services and that proxy will have its own url ( which translates to a base domain )

> So actually I wanted to even more separate the URLs than to group them. Do we care?

Indeed i care, this is creating problems while trying to remove the hardcoded & url dependent parts from koding ( ie: the things that i fixed while trying to make koding open source )

> This is to fix hacks like this one:

Infact they should not exist in the code base, you could use `customDomain.local` https://github.com/koding/koding/blob/master/config/main.dev.coffee#L65   Could be helpful for future if we have PODs/services, because localhost you know